### PR TITLE
edit ASC speck documentation

### PR DESCRIPTION
**Pull request by jenna-tomkinson**
This PR includes edited documentation regarding the wave 2 data from Plate 1. It adds more details on the treatments used + adding another figure.

### CODE REVIEW

File: Plate_1_data/1.wave2_data/0.download_data/README.md [hunk 1]
```diff
@@ -1,23 +1,4 @@
...
-- Mito_641_narrow -> Mitochondria
-- Alexa 700_narrow -> ASC
-
-![wave2_platemap](figures/wave2_platemap_fig.png)
-
-Since Alexa 488 and Alexa 700_narrow are staining for different things based on the condition, the metadata for these two channels will include both conditions 7 and 8 (e.g. Alexa 488 = ASC_nothing). 
-Since the condition is not included in the metadata `Index.idx.xml` file, it will be added to the final extracted single cell csv file manually.
-
-**Note:** The original name of the plate (or collect of wells in a plate) had to be changed due to the inclusion of spaces in the name causing issues downstream.
+In this module, we download the images from this specific wave of the plate to analyze.
```
Comment (MattsonCam) [inline Plate_1_data/1.wave2_data/0.download_data/README.md:3]: Is the wave of the plate here referring to the version of the plate?

File: Plate_1_data/1.wave2_data/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# ASC Bleedthrough Assessment - Wave 2
+
+In this wave of Plate 1 data, we have images from two different conditions with four channels, totalling 4,608 images.
+
+The conditions are as follows:
+
+For **condition 7**:
+- HOECHST 33342 -> Nuclei
```
Comment (MattsonCam) [inline Plate_1_data/1.wave2_data/README.md:8]: May consider adding a verbal description about how this mapping should be interpreted.

File: Plate_1_data/1.wave2_data/README.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# ASC Bleedthrough Assessment - Wave 2
```
Comment (MattsonCam) [inline Plate_1_data/1.wave2_data/README.md:1]: May also consider adding a verbal description of what this means and the overall goal of this assessment.

File: Plate_1_data/1.wave2_data/figures/wave2_treatment_fig.png
Comment (MattsonCam) [inline Plate_1_data/1.wave2_data/figures/wave2_treatment_fig.png:1]: Maybe I missed this, but why are all of the other sections greyed out? May consider mentioning this.

**Review by MattsonCam — APPROVED** [review]
LGTM, there are a few parts that I don't understand, that I tried addressing in comments. Good work overall @jenna-tomkinson !

File: Plate_1_data/1.wave2_data/0.download_data/README.md (hunk 1 above)
Comment (jenna-tomkinson) [reply Plate_1_data/1.wave2_data/0.download_data/README.md:3]: The wave refers to the section of the plate.

File: Plate_1_data/1.wave2_data/README.md (hunk 2 above)
Comment (jenna-tomkinson) [reply Plate_1_data/1.wave2_data/README.md:8]: I am not 100% sure what you mean here by mapping. In this section, it is going over the stains and what is being stained for in each condition.

File: Plate_1_data/1.wave2_data/README.md (hunk 3 above)
Comment (jenna-tomkinson) [reply Plate_1_data/1.wave2_data/README.md:1]: Added a sentence with quick description here. Good point!

File: Plate_1_data/1.wave2_data/figures/wave2_treatment_fig.png
Comment (jenna-tomkinson) [reply Plate_1_data/1.wave2_data/figures/wave2_treatment_fig.png:1]: In the above figure, it boxes a specific portion of the plate. In this figure you are referencing, I greyed out the rest of the plate as to highlight the part of this plate and the different treatments being used.

**Pull request merged**